docs/contributors/code/auto-cherry-picking.md

```diff
@@ -0,0 +1,73 @@
+# Cherry-picking automation
+
+The cherry-picking automation makes cherry-picking PRs into WordPress and Gutenberg release branches a breeze.
```

Comment:
This document is also linked from the section that covers cherry-picking bugs for the Gutenberg plugin releases. As far as I know, we don't have a label for plugin releases, so we should create one as part of the process.

It would also be helpful to make it clear at the top of the document that, by default, it works with the current major WordPress release. Alternatively, we could add the logic that detects the label based on the name of the branch:
- `release/X.Y` - plugin release
- `wp/X.Y` - WP release

Reply:
There's nothing major WP release specific in here – it works for the current git branch. I'll clarify that further.

Reply:
The autodetection logic would be really nice, too! I'll leave it as a TODO note for anyone who wishes to pick that up

Reply:
I think the confusion would get resolved with autodetection of the label that should be scanned.

Anyway, it's fine to leave a note with some examples how to approach that based on the scenario.